cmd/gb-list/package_view.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import "github.com/constabulary/gb"
+
+type PackageView struct {
```

Comment:
Did this come from the Go sources ? If more than a little of it was copied, best to also copy the copywrite statement from the top of the file.